setup.cfg

```diff
@@ -41,9 +41,9 @@ console_scripts =
     pyproject-build = build.__main__:entrypoint
 
 [options.extras_require]
-doc =
+docs =
```

Comment:
Why?

Reply:
I suppose because we speak of "docs" in the commit messages and the folder itself is called "docs".

Reply:
(I won't split the commits up if that's ok but I'll keep it in mind for future PRs.)

Reply:
Yeah, that's fine.